cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +794,56 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+const char* gdv_fn_concat_ws_utf8(int64_t context, const char* separator,
+                                  int32_t separator_len, const char* data,
+                                  int32_t data_len, int32_t* out_len) {
+  if (data_len <= 0) {
+    gdv_fn_context_set_error_msg(context, "Data can not be null.");
+    *out_len = 0;
+    return "";
+  }
+
+  std::string concat = "";
+  std::string data_str(data, data_len);
+  std::string sep(separator, separator_len);
+
+  size_t pos_str = 0;
+  std::string token;
+  while ((pos_str = data_str.find(',')) != std::string::npos) {
```

Comment:
The Hive function documentation has nothing about splitting the input by ','. Why are we doing this?

Reply:
It was to simulate a variadic function with an undefined number of arguments, but I corrected it to have a fixed number of parameters